chart/templates/workers/worker-kedaautoscaler.yaml

```
@@ -37,10 +37,10 @@ spec:
   scaleTargetRef:
     kind: {{ ternary "StatefulSet" "Deployment" .Values.workers.persistence.enabled }}
     name: {{ .Release.Name }}-worker
-  pollingInterval:  {{ .Values.workers.keda.pollingInterval }}   # Optional. Default: 30 seconds
-  cooldownPeriod: {{ .Values.workers.keda.cooldownPeriod }}    # Optional. Default: 300 seconds
+  pollingInterval:  {{ .Values.workers.keda.pollingInterval }}   # Optional. Default: 5 seconds
+  cooldownPeriod: {{ .Values.workers.keda.cooldownPeriod }}    # Optional. Default: 30 seconds
```

```suggestion
  pollingInterval:  {{ .Values.workers.keda.pollingInterval }}
  cooldownPeriod: {{ .Values.workers.keda.cooldownPeriod }}
```

Let's actually just remove it completely - we don't really do this elsewhere and they are just asking to become out of sync.

---

Done, thanks for the suggestions : )